servers/republik/modules/crowdfundings/lib/scheduler/owners.js

```diff
@@ -97,10 +106,24 @@ const getBuckets = async ({ now }, { pgdb }) => {
       )
```

Comment:
`prolongBeforeDate` might be wrong. How can you be sure it's the date for the membershipType you are looking for? I think for this to be correct you need to pass the type to `getProlongBeforeDate` and only check the desired types there. If only we knew these requirements before...